src/overlays/actors/ovl_En_Niw_Girl/z_en_niw_girl.c

```diff
@@ -21,21 +31,226 @@ const ActorInit En_Niw_Girl_InitVars = {
     (ActorFunc)EnNiwGirl_Update,
     (ActorFunc)EnNiwGirl_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Girl/EnNiwGirl_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Girl/EnNiwGirl_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 10, 30, 0, { 0, 0, 0 } },
+};
+
+static Vec3f sConstVec3f = { 0.2f, 0.2f, 0.2f };
+
+Gfx* D_80AB99D8[] = { 0x06004178, 0x06004978, 0x06005178, 0x00000000, 0x00000000, 0x00000000 };
+
+extern SkeletonHeader D_06009948;
+extern AnimationHeader D_06000378;
+extern AnimationHeader D_06009C78;
+
+void EnNiwGirl_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnNiwGirl* this = THIS;
+    s32 pad;
+    Vec3f tempSrc;
+    Vec3f tempDst;
```

Comment:
I suggest just naming these vec1 and vec2 if we don't know what they correspond to. (src and dst doesn't really makes sense in this context)
```suggestion
    Vec3f vec1;
    Vec3f vec2;
```